src/Map/src/Bridge/Google/src/Option/ControlPosition.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Bridge\Google\Option;
+
+/**
+ * Identifiers used to specify the placement of controls on the map.
+ * Controls are positioned relative to other controls in the same layout position.
+ * Controls that are added first are positioned closer to the edge of the map.
+ * Usage of "logical values" (see https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) is recommended
+ * in order to be able to automatically support both left-to-right (LTR) and right-to-left (RTL) layout contexts.
+ *
+ * @see https://developers.google.com/maps/documentation/javascript/reference/control#ControlPosition
+ *
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+enum ControlPosition: int
+{
+    /**
+     * Equivalent to {@see self::BOTTOM_CENTER} in both LTR and RTL.
+     */
+    case BLOCK_END_INLINE_CENTER = 24;
+
+    /**
+     * Equivalent to {@see self::BOTTOM_RIGHT} in LTR, or {@see self::BOTTOM_LEFT} in RTL.
+     */
+    case BLOCK_END_INLINE_END = 25;
+
+    /**
+     * Equivalent to {@see self::BOTTOM_LEFT} in LTR, or {@see self::BOTTOM_RIGHT} in RTL.
+     */
+    case BLOCK_END_INLINE_START = 23;
+
+    /**
+     * Equivalent to {@see self::TOP_CENTER} in both LTR and RTL.
+     */
+    case BLOCK_START_INLINE_CENTER = 15;
+
+    /**
+     * Equivalent to {@see self::TOP_RIGHT} in LTR, or {@see self::TOP_LEFT} in RTL.
+     */
+    case BLOCK_START_INLINE_END = 16;
+
+    /**
+     * Equivalent to {@see self::TOP_LEFT} in LTR, or {@see self::TOP_RIGHT} in RTL.
+     */
+    case BLOCK_START_INLINE_START = 14;
+
+    /**
+     * Elements are positioned in the center of the bottom row. Consider using
+     * {@see self::BLOCK_END_INLINE_CENTER} instead.
+     */
+    case BOTTOM_CENTER = 11;
+
+    /**
+     * Elements are positioned in the bottom left and flow towards the middle.
+     * Elements are positioned to the right of the Google logo. Consider using
+     * {@see self::BLOCK_END_INLINE_START} instead.
+     */
+    case BOTTOM_LEFT = 10;
+
+    /**
+     * Elements are positioned in the bottom right and flow towards the middle.
+     * Elements are positioned to the left of the copyrights. Consider using
+     * {@see self::BLOCK_END_INLINE_END} instead.
+     */
+    case BOTTOM_RIGHT = 12;
```

Comment:
As Google describe those as "legacy" ... do we need to implement them ? This class  / doc could be lighter whithout, no ?

Reply:
I wanted to be iso with "real" JS options, but I believe we can remove them yea